projects/kit/directives/unfinished-validator/unfinished-validator.directive.ts

```diff
@@ -14,6 +14,9 @@ import {tuiCreateUnfinishedValidator} from '@taiga-ui/kit/validators';
     ],
 })
 export class TuiUnfinishedValidatorDirective implements Validator {
+    @Input('tuiUnfinishedValidator')
+    message: string | null = null;
```

Comment:
It should support boolean attribute `<tui-input-date tuiUnfinishedValidator>`. Does it still work like that?

Reply:
fixed